src/System.Time/System/Time.cs

```diff
@@ -1681,40 +1662,37 @@ private static string NormalizeTimeFormat(string format)
             // standard formats
             if (format.Length == 1)
             {
-                // pass-through formats
-                if ("Tt".Contains(format))
+                switch (format)
                 {
```

Comment:
Same comment as for Date. switch(format[0])

Reply:
Fixed.  Thanks.